tools/autograd/gen_variable_type.py

```diff
@@ -1206,10 +1206,11 @@ def emit_increment_version():
     body.append(declare_returned_variables)
 
     body.append(emit_call(env, tie_return_values))
+    if strategy == 'use_derived':
+        body.extend(emit_increment_version())
```

Comment:
So the idea here is, if the op in question is composite, we DON'T emit the increment version (so that the constituent pieces can take care of it)